images/br-federation-manager/Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM alpine:3.14
+FROM pingcap/tools-base:v1
```

Comment:
there is `Dockerfile.e2e` for br-federation-manager too.

Reply:
added